docs/R-IOC/r07_data_import_export.md

```diff
@@ -0,0 +1,324 @@
+<!-- ## Data Import and Export -->
+
+We can deal with different formats of data with R, such as
+the text files (.csv, .tsv, .txt), the Excel files (.xlx, .xlsx), and the R data file formats (.RDS, .RData).
+It is also possible to read or write other program-speficied formats, for example
+"SAS", "SPSS" or "Minitab" files.
+
+There are R functions which allow users to download files from the Internet, for example the `download.file()` from the `utils` package or the similar function `curl_download()` from the <code>[curl](https://cran.r-project.org/web/packages/curl/index.html)</code> package.
+
+```r
+download.file(
+  url = "https://ftp.ebi.ac.uk/pub/databases/spot/pgs/scores/PGS000841/ScoringFiles/PGS000841.txt.gz",
+    # example of the polygenic risk score file for the trait BMI
+  destfile = "path/to/out_dir"
+)
+
+# install.packages("curl")
+curl::curl_download(
+  url = "http://url_to_wanted_file",
+  destfile = "path/to/out_dir"
+)
+```
+
+## Importing Data
+
+In order to avoid error when you import the file,
+it's suggested to name the data columns following the naming conventions (see [Best Practices](r04_bestpractices.md) section).
+
+### Read Text Files (.csv/.tsv/.txt)
+
+With the functions from the basic package "utils":
+
+```r
+## read a comma separated file ".csv"
+my_csv <- read.csv(
+  file = "path/to/my_file.csv",
+  header = TRUE,    # whether the file has a colnames in the 1st row
+  sep = ",",        # the character used in the file to separate columns
+  quote = "\"",     # the character(s) for quotes
```

Comment:
I'll specify the use of "\" character to "escape" the double quote because it's not very clear and learners are not used with regular expression

Reply:
an explanation about the use of backslash is added ;)